src/NNLSSolverInterface.jl

```diff
@@ -0,0 +1,102 @@
+module NNLSSolverInterface
+
+export QPSolver
```

Comment:
It's not exported from NNLS (just from the submodule), but you're still correct. I think just `NNLSSolver` is probably best (since we might eventually add support for LPs)